src/main/kotlin/dev/supergrecko/kllvm/core/typedefs/Builder.kt

```diff
@@ -106,6 +108,27 @@ public class Builder public constructor(context: Context = Context.getGlobalCont
             )
         )
     }
+
+    public fun buildAlloca(type: Type, name: String): InstructionValue {
+        return InstructionValue(LLVM.LLVMBuildAlloca(ref, type.ref, name))
+    }
+
+    public fun buildLoad(ptr: Value, name: String): InstructionValue {
+        return InstructionValue(LLVM.LLVMBuildLoad(ref, ptr.ref, name))
+    }
+
+    public fun buildStore(value: Value, toPointer: Value): InstructionValue {
+        return InstructionValue(LLVM.LLVMBuildStore(ref, value.ref, toPointer.ref))
+    }
+
+    public fun buildExtractValue(aggVal: Value, index: Int, name: String): InstructionValue {
+        return InstructionValue(LLVM.LLVMBuildExtractValue(ref, aggVal.ref, index, name))
+    }
+
+    public fun buildStructGEP(pointer: Value, index: Int, name: String): InstructionValue {
+        return InstructionValue(LLVM.LLVMBuildStructGEP(ref, pointer.ref, index, name))
+    }
```

Comment:
I think we want to create a nicer API for the builder in the future but this is good enough for now.